.eslintrc

```diff
@@ -0,0 +1,100 @@
+{
+	"env": {
+		"browser": true,
+		"node": true,
+	},
+	"extends": "eslint:recommended",
+	"globals": {
+		"Prism": "readonly",
+	},
+	"parserOptions": {
+		"ecmaVersion": 5,
+	},
+	"root": true,
+	"rules": {
+		"comma-dangle": [
+			"warn",
+			"only-multiline",
+		],
+//		"indent": [
+//			"error",
+//			"tab",
+//			{
+//				"SwitchCase": 1,
+//				"outerIIFEBody": 0,
+//			}
+//		],
+		"linebreak-style": [
+			"error",
+			"unix",
+		],
+		"no-console": "off",
+		"no-empty": "off",
+		"no-inner-declarations": "off",
+		"no-mixed-spaces-and-tabs": "error",
+		"no-redeclare": "off",
+		"no-sparse-arrays": "off",
+		"no-unused-vars": [
+			"warn",
+			{
+				"args": "none",
+				"varsIgnorePattern": "^_",
+			}
+		],
+		"no-useless-escape": "off",
+//		"quotes": [
+//			"warn",
+//			"single",
+//			{
+//				"avoidEscape": true,
+//			}
+//		],
+		"semi": [
+			"error",
+			"always",
+		],
+	},
+	"overrides": [
+		{
+			"files": [
+				"gulpfile.js",
+			],
+			"env": {
+				"es6": true,
+			},
+			"parserOptions": {
+				"ecmaVersion": 2015,
+			},
+			"rules": {
+				"comma-dangle": [
+					"error",
+					"always-multiline",
```

Comment:
Why did you change the behavior here?

Reply:
Because `gulpfile.js` already has traling commas in&nbsp;most places.

Reply:
But do we really want to force people to use commas because of that?
Also, isn't ESLint support to make the code style consistent?

Reply:
I eventually want to enforce <code>[comma‑dangle](https://eslint.org/docs/rules/comma-dangle): "always‑multiline"</code> everywhere.

Reply:
I can see why you want this but it literally requires us to change (almost) every language definition.

I'm personally quite a fan of `"only-multiline"`.
We should get some more opinions on this.

@mAAdhaTTah @Golmote @LeaVerou

Reply:
I'm not a fan of dangling commas. I see the benefit in multiline cases, but they are also less natural and I think optimizing our code for tooling is a slippery slope. But my objection is not so strong that is should hold this up if the rest of you agree. If you don't however, do count it as an objection.

Reply:
I kinda like dangling commas, but in this case, I think the churn on the language definitions makes it not worth it.

I'd also prefer not to have a different rule for the `gulpfile.js` but instead make the changes in the `gulpfile.js` to bring it in line with the rest of the codebase.

Reply:
The thing is that some code is written as if [`comma‑dangle`](https://eslint.org/docs/rules/comma-dangle) was `"always‑multiline"` and other as if it was `"only‑multiline"`.